src/components/Form/TourForm.tsx

```diff
@@ -1,12 +1,75 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Debrief from './Debrief'
 import Ride from './Ride'
-import Plan from './Plan'
+import { Plan } from './Plan'
+import { TextField } from './TextField'
+
+interface TourFormProps {
+  userId: number,
+}
+
+export interface TourFormState {
+  location: string,
+  date: string,
+  hazardWeather: string,
+  hazardAvalanche: string,
+  hazardSummary: string,
+  routePreview: string,
+  routeAlternative: string,
+  emergencyPlan: string,
+  debriefConditions: string,
+  debriefDecisions: string,
+  debriefPlan: string,
+}
+
+export const TourForm: React.FC<TourFormProps> = ({ userId }) => {
+  const [state, setState] = useState<TourFormState>({
+    location: '',
+    date: Date.now().toString(),
+    hazardWeather: '',
+    hazardAvalanche: '',
+    hazardSummary: '',
+    routePreview: '',
+    routeAlternative: '',
+    emergencyPlan: '',
+    debriefConditions: '',
+    debriefDecisions: '',
+    debriefPlan: ''
+  })
+
+  const renderTextInputs = (fields: string[]): JSX.Element[] => {
+    return fields.map(field=> {
+
+      return (
+        <TextField
+          value={state[field as keyof TourFormState]}
+          updateForm={e => setState({ ...state, [field]: e.target.value})}
+        />
+      )
+    })
+  }
+
+  const isChecked = (fields: string[]) => {
+    return !fields.find(field => state[field as keyof TourFormState] === '')
+  }
 
-export default function TourForm() {
   return (
     <div>
-      
+      <input
+        type='date'
+        value={state.date}
+        onChange={e => setState({ ...state, date: e.target.value})}
+        min={Date.now()}
```

Comment:
Did this actually work or is it something we will have to go back to later?

Reply:
We need to come back to it later.